packages/components/note/src/Note.tsx

```diff
@@ -108,7 +108,10 @@ export const Note = React.forwardRef<HTMLElement, ExpandProps<NoteProps>>(
             onClick={onClose}
             testId={`${testId}-close`}
             aria-label="Dismiss"
-            className={styles.close}
+            className={cx(
+              styles.close({ title }),
+              className,
```

Comment:
Is it intended to merge with `className` here? It's already done on the container element.

Reply:
oops, yeah, sorry, thought it wasn’t passed.